charts/cryostat/tests/cryostat_deployment_test.yaml

```diff
@@ -651,90 +651,60 @@ tests:
           path: spec.template.spec.containers[?(@.name=='cryostat-jfr-datasource')].imagePullPolicy
           value: "IfNotPresent"
 
-  - it: should add volume mounts for declarative event templates
+  - it: should add volume mounts for declarative automated rules
     set:
-      core.config.eventTemplates.configMapNames: ['a', 'b']
+      core.config.rules.configMapNames: ['a', 'b']
```

Comment:
Was this test case merged correctly? It looks like it's deleting some coverage of other features.